cleanup-passes/travis.ts

```diff
@@ -26,18 +26,35 @@ type TravisEnv = {global?: string[], matrix?: string[]};
 
 interface TravisConfig {
   before_script?: string[];
+  install?: string[];
   addons?: {
     firefox?: string | number,
+    chrome?: string | number,
     sauce_connect?: boolean,
     apt?: {
       packages?: string[],
-      sources?: string[]
-    }
+      sources?: string[],
+    },
   };
-  dist?: string,
-  sudo?: 'false' | 'required',
+  script?: string[];
+  dist?: string;
+  sudo?: 'false' | 'required';
   env?: TravisEnv;
-  node_js?: string | number;
+  node_js?: string | number | string[];
+  cache?: {
+    directories?: string[],
+  };
+}
+
+function arraysEqual(a: String[], b: String[]): Boolean {
```

Comment:
I think the StackOverflow policy is to link to the original snippet in a comment, to properly credit the original author.